.gitignore

```diff
@@ -40,3 +40,7 @@ Makefile.in
 /missing
 /m4
 /test-driver
+
+# Meson
+/subprojects
+/build
```

Comment:
It does not do it for the generated parts of subprojects/, in part because that could theoretically clash with existing .gitignore files extracted from a tarball, in part because that has weird interactions with wrap-git, and in part because for all we know, some projects do want to check subprojects into git. I think doing so is weird and inadvisable, but I'm not sure we should stop them.

The generated build directories do, because we know that meson always creates it from scratch and it only contains our own build artifacts which *cannot* be checked into git. Note also that we added this functionality in 0.57.0, but you depend on >=0.60 so that is a moot point.